Assets/Scripts/PlayerCollision.cs

```diff
@@ -5,16 +5,15 @@ public class PlayerCollision : MonoBehaviour
     //reference to our movement script.
     public PlayerMovement movement;
 
-    //will be called on collision if object as a rigid body and a collider
+    //will be called on collision if object has a rigid body and a collider
     void OnCollisionEnter(Collision collisionInfo)
     {
 
         if (collisionInfo.collider.tag == "Obstacle")
         {
-            movement.enabled = false;
+            movement.DisableMovement();
```

Comment:
Real minor but this method is also called within `FindObjectOfType<GameManager>().EndGame();`

Reply:
good catch